Duplicati/GUI/Duplicati.GUI.TrayIcon/Program.cs

```diff
@@ -174,7 +177,10 @@ public static void RealMain(string[] _args)
 
             if (options.TryGetValue(HOSTURL_OPTION, out url))
                 serverURL = new Uri(url);
-            
+
+            System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11;
+            new Duplicati.Library.Modules.Builtin.HttpOptions().Configure(options);

```

Comment:
The `Duplicati.Library.Modules.Builtin.HttpOptions` class is meant to be disposed.
It is a bit of a dangling setup to use it here, as it may be overridden later by a backup.

What about setting the protocol on the request here instead:
https://github.com/duplicati/duplicati/blob/master/Duplicati/GUI/Duplicati.GUI.TrayIcon/HttpServerConnection.cs#L357